app/segments/drill/[id]/submission/input.js

```diff
@@ -341,7 +346,8 @@ function createOutputData(drillInfo, inputValues, attemptShots, uid, did) {
   console.log("outputShotData", outputShotData);
 
   //get the time stamp
-  const timeStamp = Date.now();
+  // convert from ms to seconds to match db_spec.jsonc
+  const timeStamp = Date.now() / 1000;
```

Comment:
Let's do miliseconds

Reply:
It should be fixed now.  I also had to change `const date = formatDate(curr.assignedTime * 1000);` in Plan file to avoid errors, that was only thing I missed earlier.

Newly submitted drill attempt submissions will have the correct date formatting, but old submissions may have the date listed as 1970 (start of Unix time).

![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/82061589/6c71021a-cd49-4735-93fa-5d53ecff4537)

Reply:
I think something was mentioned about drill description predicate (for invalidate queries) being wrong, but it looked fine to me after testing it again.